internal/datastore/crdb/tx.go

```diff
@@ -129,42 +96,7 @@ func executeWithResets(ctx context.Context, conn conn, txOptions pgx.TxOptions,
 	return
 }
 
-// executeWithRetries executes the transaction fn and attempts to retry up to maxRetries.
-// adapted from https://github.com/cockroachdb/cockroach-go/blob/05d7aaec086fe3288377923bf9a98648d29c44c6/crdb/tx.go#L95
-func executeWithRetries(ctx context.Context, currentTx pgx.Tx, fn transactionFn, maxRetries int) (retries int, err error) {
-	releasedFn := func(tx pgx.Tx) error {
-		if err := fn(tx); err != nil {
-			return err
-		}
-
-		// RELEASE acts like COMMIT in CockroachDB. We use it since it gives us an
-		// opportunity to react to retryable errors, whereas tx.Commit() doesn't.
-
-		// RELEASE SAVEPOINT itself can fail, in which case the entire
-		// transaction needs to be retried
-		if _, err := tx.Exec(ctx, sqlReleaseSavepoint); err != nil {
-			return err
-		}
-		return nil
-	}
-
-	var i int
-	for i = 0; i <= maxRetries; i++ {
-		if err = releasedFn(currentTx); err != nil {
-			if retriable(ctx, err) {
-				if _, retryErr := currentTx.Exec(ctx, sqlRollback); retryErr != nil {
-					return i, RetryError{err: retryErr}
-				}
-				continue
-			}
-			return i, err
-		}
-		return i, nil
-	}
-	return i, errors.New(errReachedMaxRetry)
-}
-
-// resetExecution attempts to rollback the given tx, begins a new tx with a new connection, and creates a savepoint.
+// resetExecution attempts to rollback the given tx and begins a new tx with a new connection.
 func resetExecution(ctx context.Context, conn conn, tx pgx.Tx, txOptions pgx.TxOptions) (newTx pgx.Tx, err error) {
 	log.Info().Msg("attempting to initialize new tx")
```

Comment:
I know this here before, but now that it's getting called on revision and namespace reads, shouldn't this be a higher log level than `Info`?